Add More Architectures

ax3l opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged): not needed, only new archs
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

ax3l committed:
Add More Architectures

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ax3l commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.9.0, and conda-forge-pinning 2021.03.07.18.51.23

ax3l committed:
-macOS ARM

Skip for now, needs proper upstream treatment:
```
-- Checking BLAS complex return type
CMake Error: TRY_RUN() invoked in cross-compiling mode, please set the following cache variables appropriately:
   run_result (advanced)
   run_result__TRYRUN_OUTPUT (advanced)
For details see /Users/runner/miniforge3/conda-bld/blaspp_1615193440206/work/build/TryRunResults.cmake
CMake Error at cmake/BLASConfig.cmake:182 (message):
  Error - Cannot detect zdotc return value.  Please check the BLAS
  installation.
Call Stack (most recent call first):
  CMakeLists.txt:303 (include)
```

ax3l commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.9.0, and conda-forge-pinning 2021.03.07.18.51.23
